docker/addons/certs/docker-compose.yml

```
@@ -70,6 +70,7 @@ services:
       MF_AUTH_GRPC_URL: ${MF_AUTH_GRPC_URL}
       MF_AUTH_GRPC_TIMEOUT: ${MF_AUTH_GRPC_TIMEOUT}
       MF_CERTS_VAULT_HOST: ${MF_CERTS_VAULT_HOST}
+      MF_CERTD_INSTANCE_ID: ${MF_CERTD_INSTANCE_ID}
     volumes:
```

I hope here we should have  `MF_CERTS`


```suggestion
      MF_CERTS_INSTANCE_ID: ${MF_CERTS_INSTANCE_ID}
```